### PR TITLE
[COGNITION]: Education STEM Update - Step 6 has the same page label twice in Education Benefit chapter #830

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/830

## Testing done
- local testing

## Screenshots
Screen Shot 2019-08-08 at 1 43 36 PM

Screen Shot 2019-08-08 at 1 43 52 PM

Screen Shot 2019-08-08 at 1 44 21 PM

## Acceptance criteria
- [ ] Title is changed to *Education benefit selection* for the page where the user selects which benefit they're using or has used most recently
- [ ] Title is changed to *The Rogers STEM Scholarship* for the page which asks whether the user is applying for the scholarship and, if yes, displays conditional questions about their course of study

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
Approved by QA

**Review by accbjt — APPROVED** [review]
LGTM

File: src/platform/forms/tests/forms.unit.spec.js [hunk 1]
```diff
@@ -62,29 +65,68 @@ const excludedForms = new Set([
...
+
+      describe('chapters:', () => {
+        it('should have chapters', () => {
+          expect(form.chapters).to.be.an('object');
+        });
+      });
+
+      describe('defaultDefinitions:', () => {
+        it('should have defaultDefinitions', () => {
+          expect(form.defaultDefinitions).to.not.be.a('undefined');
```
Comment (cvalarida) [inline src/platform/forms/tests/forms.unit.spec.js]:
```suggestion
          expect(form.defaultDefinitions).to.not.be.undefined;
```

**Pull request merged**